tasks/src/robocorp/tasks/cli.py

```diff
@@ -30,7 +20,9 @@
     sys.coinit_flags = _COINIT_MULTITHREADED  # type:ignore
 
 # Just importing is enough to register the commands
-from . import _commands  # noqa
+from . import _commands, inject_truststore  # noqa
+
+inject_truststore()
```

Comment:
Safer to keep the utility implementation here and import straight from `robocorp.tasks.cli` in **browser**. (if there's no other limitation I'm unaware of)